Content.Shared/Access/Systems/AccessReaderSystem.cs

```diff
@@ -399,6 +401,9 @@ private void LogAccess(Entity<AccessReaderComponent> ent, EntityUid accessor)
             && idCard.Comp is { BypassLogging: false, FullName: not null })
             name = idCard.Comp.FullName;
 
+        if (TryComp<NameIdentifierComponent>(accessor, out var nameIdentifier) && HasComp<BorgChassisComponent>(accessor))
+        name = nameIdentifier.FullIdentifier;
+
```

Comment:
the indentation of name is wrong, also put this check before the idCard one as that one takes priority

I also see no reason to limit this to borgs so remove that check